src/nibetaseries/cli/run.py

```diff
@@ -82,6 +82,9 @@ def get_parser():
                                     'spm + derivative + dispersion'],
                            help='convolve your regressors '
                                 'with one of the following hemodynamic response functions')
+    proc_opts.add_argument('--fir-delays', default=None,
```

Comment:
Inside `main`, we should add a check to make sure that if `opts.hrf_model ==  'fir'` then `opts.fir_delays` should not be `None`.

Reply:
or you can follow the strategy I used [here](https://github.com/HBClab/NiBetaSeries/pull/213/files#diff-177742e0b69cb7dc3d6ba7782e9a2293R60) to make the command line arguments conditionally dependent.

Reply:
Okay, I added the check in `main`. Let me know if you'd prefer it implemented another way.

Reply:
I like the check you added in main, thanks!